DB내에 중복된 데이터 검출 및 제거

JDW611 opened the pull request:
- 가게 이름이 같을 때 위도, 경도도 같은 경우 중복으로 판단하고 데이터 제거

lightpurple committed:
Merge pull request #1 from team-culfare/develop

Develop

lightpurple committed:
Update README.md

lightpurple committed:
Merge pull request #2 from team-culfare/develop

V1.1.0

JDW611 committed:
DB 중복 제거 기능 개발

- 가게이름이 같을 때 위도, 경도도 같은 경우 중복으로 판단하고 제거

lightpurple commented on src/dedup/deduplication.ts:
```diff
@@ -0,0 +1,28 @@
+import { QueryTypes, Sequelize } from "sequelize";
+import { mysql } from "../lib/mysql";
+
```
이 파일은 dedup 라는 디렉터리 보다는 src/lib 안에 들어가는 게 좋을 거 같습니다.

lightpurple commented on src/dedup/deduplication.ts:
```diff
@@ -0,0 +1,28 @@
+export function deleteDuplicateStores() {
+    try {
+        mysql.query(
+            `
```
취향차이 일수도 있는데 쿼리문을 변수에 담고 mysql.query(변수명) 이런식으로 진행해도 괜찮을 거 같아요.

lightpurple commented on src/dedup/deduplication.ts:
```diff
@@ -0,0 +1,28 @@
+
+export function deleteDuplicateStores() {
+    try {
+        mysql.query(
```
async / await 구문 추가해주세요.

lightpurple commented on src/dedup/deduplication.ts:
```diff
@@ -0,0 +1,28 @@
+            }
+        );
+
+        console.log("Duplicate stores deleted successfully!!");
```
로깅은 console.log 보다 logger.log나 logger.debug를 사용해주세요.

JDW611 committed:
Pull requests 피드백 반영

- 파일 디렉터리 경로 재설정
- 쿼리문을 변수에 담고 해당 변수를 호출하는 방식으로 변경
- async / await 구문 추가
- console.log 대신 logger.log, logger.debug 사용으로 변경

lightpurple commented on src/lib/deduplication.ts:
```diff
@@ -0,0 +1,24 @@
+
+        logger.log("Duplicate stores deleted successfully!!");
+    } catch (error) {
+        logger.debug("Error deleting duplicate stores:", error);
```
logger 모듈 보면 error 옵션이 있습니다. 다음엔 에러 캐치시에는 debug 보단 error 옵션을 사용하는게 좋아보여요.
